src/main/config/wikipedia-feed.properties

```diff
@@ -23,22 +23,7 @@ job.name=wikipedia-feed
 yarn.package.path=file://${basedir}/target/${project.artifactId}-${pom.version}-dist.tar.gz
 
 # Task
-task.class=samza.examples.wikipedia.task.WikipediaFeedStreamTask
-task.inputs=wikipedia.#en.wikipedia,wikipedia.#en.wiktionary,wikipedia.#en.wikinews
-
-# Serializers
-serializers.registry.json.class=org.apache.samza.serializers.JsonSerdeFactory
-
-# Wikipedia System
-systems.wikipedia.samza.factory=samza.examples.wikipedia.system.WikipediaSystemFactory
-systems.wikipedia.host=irc.wikimedia.org
-systems.wikipedia.port=6667
-
-# Kafka System
-systems.kafka.samza.factory=org.apache.samza.system.kafka.KafkaSystemFactory
-systems.kafka.samza.msg.serde=json
-systems.kafka.consumer.zookeeper.connect=localhost:2181/
-systems.kafka.producer.bootstrap.servers=localhost:9092
+app.class=samza.examples.wikipedia.task.application.WikipediaFeedStreamTaskApplication
 
 # Job Coordinator
 job.coordinator.system=kafka
```

Comment:
Can you remove this (everywhere) and set the default system descriptor in the TaskApplication instead?

Reply:
Can you also update teh comment below if necessary. Seems outdated.